easybuild/easyconfigs/o/OpenCV/OpenCV-4.6.0-foss-2022a-contrib.eb

```diff
@@ -0,0 +1,95 @@
+name = 'OpenCV'
+version = '4.6.0'
+versionsuffix = '-contrib'
+
+# the hash is version dependent! see 3rdparty/ippicv/ippicv.cmake
+local_ippicv_hash = 'a56b6ac6f030c312b2dce17430eef13aed9af274'
+
+homepage = 'https://opencv.org/'
+description = """OpenCV (Open Source Computer Vision Library) is an open source computer vision
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products.
+ Includes extra modules for OpenCV from the contrib repository."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/%(namelower)s/%(namelower)s/archive/'],
+        'download_filename': '%(version)s.zip',
+        'filename': SOURCELOWER_ZIP
+    },
+    {
+        'source_urls': ['https://github.com/%(namelower)s/opencv_contrib/archive/'],
+        'download_filename': '%(version)s.zip',
+        'filename': '%(namelower)s_contrib-%(version)s.zip'
+    },
+    {
+        'source_urls': ['https://raw.githubusercontent.com/opencv/opencv_3rdparty/%s/ippicv' % local_ippicv_hash],
+        'filename': 'ippicv_2020_lnx_intel64_20191018_general.tgz',
+        'extract_cmd': 'cp %s %(builddir)s'
+    },
+]
+checksums = [
+    '158db5813a891c7eda8644259fc1dbd76b21bd1ffb9854a8b4b8115a4ceec359',  # opencv-4.6.0.zip
+    '588ca44e3c88ccf8e3ff7f46e966e26ff63d72607968ace83b91628d3440f8a3',  # opencv_contrib-4.6.0.zip
+    '08627fa5660d52d59309a572dd7db5b9c8aea234cfa5aee0942a1dd903554246',  # ippicv_2020_lnx_intel64_20191018_general.tgz
+]
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+]
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('zlib', '1.2.12'),
+    ('FFmpeg', '4.4.2'),
+    ('freetype', '2.12.1'),
+    ('HarfBuzz', '4.2.1'),
+    ('libjpeg-turbo', '2.1.3'),
+    ('OpenJPEG', '2.5.0'),
+    ('libpng', '1.6.37'),
+    ('LibTIFF', '4.3.0'),
+    ('libwebp', '1.2.4'),
+    ('OpenEXR', '3.1.5'),
+    ('JasPer', '2.0.33'),
+    ('Java', '11', '', True),
```

Comment:
```suggestion
    ('Java', '11', '', SYSTEM),
```

Reply:
Haha, nice, it's good to know that new test is working! :)